extensions/blocks/contact-form/index.js

```diff
@@ -90,6 +90,11 @@ const FieldDefaults = {
 			default: 100,
 		},
 	},
+	postLinkText: {
+		type: 'string',
+		source: 'html',
+		default: __( 'Submit a contact form.', 'jetpack' ),
+	},
```

Comment:
Given this is a new attribute, it should be under `attributes`. Also, when the source of an attribute is the HTML DOM, it needs a selector so it can be fetched. See the "Pay with Paypal" block as reference: https://github.com/Automattic/jetpack/blob/b7ccbb1d60d8d0a1463711836a810d1d5f012728/extensions/blocks/simple-payments/index.js#L123-L128